sportsdataverse/cfb/cfb_pbp.py

```
@@ -2953,6 +3179,400 @@ def __add_player_cols(self, play_df):
+            sack_team=pl.col("def_pos_team"),
+            interception_team=pl.col("def_pos_team"),
+            pass_breakup_team=pl.col("def_pos_team"),
+            forced_fumble_team=pl.col("def_pos_team"),
+            # Team that recovered the fumble/muff. Prefer the parsed recovering-team
+            # abbreviation; when it does not parse, fall back to the gaining team (the
+            # side opposite the fumbling team) for turnovers, or the fumbling team for
+            # own recoveries -- so a recovery is never dropped just because the team
+            # abbreviation in the text could not be matched.
+            fumble_recovery_team=pl.when(pl.col("recovery_team").is_not_null())
+            .then(pl.col("recovery_team"))
+            .when(pl.col("fumble_or_muff") == False)
```

Flake8 (7.3.0)

[error] comparison to False should be 'if cond is False:' or 'if not cond:'

(E712)